aws/table_aws_ecr_image.go

```diff
@@ -44,6 +45,13 @@ func tableAwsEcrImage(_ context.Context) *plugin.Table {
 				Description: "The sha256 digest of the image manifest.",
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "image_uri",
+				Description: "It is a unique identifier that specifies the location and version of the image.",
```

Comment:
Can you please look at the description of the repository_uri column and follow a similar style?

Reply:
updated.